win32ss/gdi/gdi32/objects/font.c

```diff
@@ -330,7 +331,12 @@ IntEnumFontFamilies(HDC Dc, const LOGFONTW *LogFont, PVOID EnumProc, LPARAM lPar
     if (INITIAL_FAMILY_COUNT < InfoCount)
     {
         RtlFreeHeap(GetProcessHeap(), 0, Info);
-        DataSize = InfoCount * sizeof(FONTFAMILYINFO);
+        DataSize64 = UInt32x32To64(InfoCount, sizeof(FONTFAMILYINFO));
+        DataSize = (LONG)DataSize64;
+        if (DataSize <= 0 || DataSize64 > LONG_MAX)
+        {
+            return 1;
```

Comment:
A DPRINT1 or something might be nice here, since it is an unexpected case. (Same with the other overflows)

Reply:
OK. DPRINT1 added.